monero-outreach-docs/translations/es/PSA/files/wallet/wallet_es.sla

```diff
@@ -60,19 +64,98 @@
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="3. Considera mantener un monedero en frío para tus XMR los cuales planees gastar o intercambiar. Piensa: Monedero "/>
+            <para LINESP="13.2"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="en frío = cuenta de ahorro, monedero caliente = cuenta corriente."/>
+            <para LINESP="13.2"/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="4. Visita foros en linea y pregunta sobre la variedad de "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="formas de como las personas mantienen segura y protegidas las llaves privadas o la semilla mnemónica. Luego escoge "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="la opción que mejor te sirva. Comienza aquí: https:// getmonero.org/community/hangouts/. Tener una estrategia "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="es mucho mejor que no tener ninguna. ¿Estas notando que "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="la investigación es importante?"/>
+            <para/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="5. ¡Sigue otro tipo de prácticas que recomendamos aquí "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="como, nunca compartas tus llaves privadas, solo utiliza casas de cambio reconocidas, actualiza tu(s) monedero(s) después de cada bifurcación de la red, no hables de cuantos XMR tienes y siempre busca aprender más!"/>
```

Comment:
"... **sólo** utiliza casas de cambio reconocidas, actualiza tu(s) monedero(s) después de cada bifurcación de la red, no hables de **cuántos** XMR tienes **y, ¡**siempre busca aprender más!

Reply:
oh...did the same suggestion...thanks

Reply:
done